fili-core/src/main/java/com/yahoo/bard/webservice/web/ApiFilter.java

```diff
@@ -113,74 +112,33 @@ public ApiFilter(
     ) throws BadFilterException {
         LOG.trace("Filter query: {}\n\n DimensionDictionary: {}", filterQuery, dimensionDictionary);
 
-        /*  url filter query pattern:  (dimension name)|(field name)-(operation)[?(value or comma separated values)]?
```

Comment:
This constructor is now dead outside of test code.
Everywhere it's currently used in test code, the filter is put into a collection anyway.  Use the helper method created for DataApiRequest so there isn't a bunch of duplicated only-slightly-different code in service to tests.  Simply change the generators for those tests to expect a collection.